Makefile

```diff
@@ -105,13 +105,15 @@ generate: gen-cli-doc gen-test-framework  ## Run all generate targets
 .PHONY: release_builds release
 
 release_builds := \
+    build/operator-sdk-$(VERSION)-aarch64-linux-gnu \
 	build/operator-sdk-$(VERSION)-x86_64-linux-gnu \
 	build/operator-sdk-$(VERSION)-x86_64-apple-darwin \
 	build/operator-sdk-$(VERSION)-ppc64le-linux-gnu \
 	build/operator-sdk-$(VERSION)-s390x-linux-gnu
 
```

Comment:
These tests need to be done manually by calling the release script and/or make release target. 
The image was created. See:

<img width="1057" alt="Screenshot 2020-03-31 at 00 13 03" src="https://user-images.githubusercontent.com/7708031/77970602-805e6000-72e4-11ea-9e77-b2cb862952a9.png">

c/c @joelanford